UnitTests/UnitTestApp.xaml.cs

```diff
@@ -23,7 +23,7 @@ namespace UnitTests
     /// <summary>
     /// Provides application-specific behavior to supplement the default Application class.
     /// </summary>
-    internal sealed partial class App : Application
+    public partial class App : Application
```

Comment:
Should it stay `sealed`?

Reply:
Does it matter? It's a unit test class. They aren't exposed anywhere

Reply:
It does not. Was curious why you put it on one file and not the Other.

Reply:
Well, I removed from.one file and not the other. Can't remember why I removed from one...